### DISCUSSION TITLE
Search across files

### DISCUSSION THREAD

**Discussion by pxlrbt** [opcodesio/log-viewer#28] [Ideas]
Often searching for issues starts with searching the right log file or moment in time, so it would be useful to search across multiple files at once.

**Comment by Sicklou** [comment]
+1 on this.

Could be very useful te be able to search one specific file by name

**Reply by arukompas** [reply]
hey @Sicklou , if no file is selected, it will search **all files**.

If you have a file selected, then it will search **that file only**.

Does this help? :)

**Reply by Sicklou** [reply]
I'm not sure we talking about the same thing. What I meant is, let's say I got 40 log files in the log viewer. I want to find one the 40 by name. I need a filter above the file list/explorer. Maybe it's better explain now ? :D

**Reply by arukompas** [reply]
@Sicklou ahh yes, now I get it! :) quickly searching through the list of files, not logs. Great idea! :) not sure when I'll have time for it, going on a long holiday soon! PRs always welcome 🙌

**Comment by arukompas** [comment]
totally forgot to let you know about this @pxlrbt , but it's now possible to search across all files! As long as no file is selected, you can start typing and it will search all files 😄
